From 2c77abd366e579c4eb700ebaae55ae0390d6acaf Mon Sep 17 00:00:00 2001
From: Fedor Lapshin <fe.lap.prog@gmail.com>
Date: Tue, 20 Feb 2024 06:25:52 +0300
Subject: [PATCH 3/3] feat(depthos): add support for posix threads

---
 gcc/config.gcc | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gcc/config.gcc b/gcc/config.gcc
index 1a30326c8..48f4072df 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -999,6 +999,11 @@ case ${target} in
 	gnu_ld=yes
 	default_use_cxa_atexit=yes
 	use_gcc_stdint=provide
+  case ${enable_threads} in
+    "" | yes | posix)
+    thread_file=posix
+    ;;
+  esac
 	;;
 *-*-*vms*)
   extra_options="${extra_options} vms/vms.opt"
-- 
2.34.1

