From d05ed7e291efc153c9d73d5bbd06c29e3e23f452 Mon Sep 17 00:00:00 2001
From: Fedor Lapshin <fe.lap.prog@gmail.com>
Date: Fri, 24 May 2024 17:29:18 +0300
Subject: [PATCH 2/2] fix multiple definitions in readline

---
 lib/readline/terminal.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/readline/terminal.c b/lib/readline/terminal.c
index 05415dc..b139357 100644
--- a/lib/readline/terminal.c
+++ b/lib/readline/terminal.c
@@ -103,9 +103,9 @@ static char *term_string_buffer = (char *)NULL;
 static int tcap_initialized;
 
 #if !defined (__linux__) && !defined (NCURSES_VERSION)
-#  if defined (__EMX__) || defined (NEED_EXTERN_PC)
+#  if defined (__EMX__) || defined(__depthos__) || defined (NEED_EXTERN_PC)
 extern 
-#  endif /* __EMX__ || NEED_EXTERN_PC */
+#  endif /* __EMX__ || __depthos__ || NEED_EXTERN_PC */
 char PC, *BC, *UP;
 #endif /* !__linux__ && !NCURSES_VERSION */
 
-- 
2.34.1

