From cb88e9af87761c7ef57b79bbd468cb7791f62829 Mon Sep 17 00:00:00 2001
From: Fedor Lapshin <fe.lap.prog@gmail.com>
Date: Thu, 9 May 2024 17:22:49 +0300
Subject: [PATCH] add depthos

---
 m4/libtool.m4 | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/m4/libtool.m4 b/m4/libtool.m4
index a3bc337..244d9ab 100644
--- a/m4/libtool.m4
+++ b/m4/libtool.m4
@@ -3009,6 +3009,17 @@ sunos4*)
   need_version=yes
   ;;
 
+depthos*)
+  version_type=linux
+  need_lib_prefix=no
+  need_version=no
+  library_names_spec='$libname$release$shared_ext$versuffix $libname$release$shared_ext$major $libname$shared_ext'
+  soname_spec='$libname$release$shared_ext$major'
+  shlibpath_var=LD_LIBRARY_PATH
+  shlibpath_overrides_runpath=no
+  hardcode_into_libs=yes
+  ;;
+
 sysv4 | sysv4.3*)
   version_type=linux # correct to gnu/linux during the next big refactor
   library_names_spec='$libname$release$shared_ext$versuffix $libname$release$shared_ext$major $libname$shared_ext'
@@ -3508,6 +3519,10 @@ haiku*)
   lt_cv_deplibs_check_method=pass_all
   ;;
 
+depthos*)
+  lt_cv_deplibs_check_method=pass_all
+  ;;
+
 hpux10.20* | hpux11*)
   lt_cv_file_magic_cmd=/usr/bin/file
   case $host_cpu in
@@ -4249,6 +4264,10 @@ m4_if([$1], [CXX], [
       # The "-static" flag exists, but is broken.
       _LT_TAGVAR(lt_prog_compiler_static, $1)=
       ;;
+    depthos*)
+      # DepthOS does not support shared libraries.
+      _LT_TAGVAR(lt_prog_compiler_pic, $1)=
+      ;;
     interix[[3-9]]*)
       # Interix 3.x gcc -fpic/-fPIC options generate broken code.
       # Instead, we relocate shared libraries at runtime.
@@ -4598,6 +4617,11 @@ m4_if([$1], [CXX], [
       enable_shared=no
       ;;
 
+    depthos*)
+      _LT_TAGVAR(lt_prog_compiler_can_build_shared, $1)=no
+      enable_shared=no
+      ;;
+
     *nto* | *qnx*)
       # QNX uses GNU C++, but need to define -shared option too, otherwise
       # it will coredump.
@@ -4847,6 +4871,9 @@ case $host_os in
   *djgpp*)
     _LT_TAGVAR(lt_prog_compiler_pic, $1)=
     ;;
+  depthos*)
+    _LT_TAGVAR(lt_prog_compiler_pic, $1)=
+    ;;
   *)
     _LT_TAGVAR(lt_prog_compiler_pic, $1)="$_LT_TAGVAR(lt_prog_compiler_pic, $1)@&t@m4_if([$1],[],[ -DPIC],[m4_if([$1],[CXX],[ -DPIC],[])])"
     ;;
-- 
2.34.1

